src/Microsoft.Identity.Client/TelemetryCore/ApiEvent.cs

```diff
@@ -70,6 +70,10 @@ public enum ApiIds
             AcquireTokenByRefreshToken = 728,
 
             AcquireTokenByAuthorizationCodeWithCodeScope = 830,
+
+            // TODO these numbers are made up
+            AcquireTokenWithBroker = 1,
```

Comment:
>1 [](start = 37, length = 1)

901 and 902 ? Santiago can likely help identify what could be used (potentially iOS has values for these already).

Reply:
going to do this later.

---
In reply to: [256630918](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/881#discussion_r256630918) [](ancestors = 256630918)